src/BuildIntegration/Microsoft.NETCore.Native.Unix.props

```diff
@@ -71,6 +76,8 @@ See the LICENSE file in the project root for more information.
       <LinkerArg Include="-lrt" Condition="'$(TargetOS)' != 'OSX'" />
       <LinkerArg Include="-licucore" Condition="'$(TargetOS)' == 'OSX'" />
       <LinkerArg Include="-shared" Condition="'$(NativeLib)' == 'Shared'" />
+      <LinkerArg Include="-lcurl" />
```

Comment:
Nit: This may be before `-shared` so that it is together with the other `-l` arguments.

Reply:
I've left them at the bottom of the includes, because in this case they were needed for the specific case of the sample app. Should we assume that they're a likely use case from now on and move them up?

Reply:
My point was to have all `-l` arguments together.

The libraries are required to make the .NET Core framework work. They are not really different from `-licucore` or `-luuid`.

Reply:
Good point. Moved.